### DISCUSSION TITLE
Beyond Compare formular missing binary link `bcompare`

### DISCUSSION THREAD

**Discussion by goerge** [Homebrew/discussions#1003] [Casks]
I wonder why the [beyond-compare](https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/beyond-compare.rb) formula does not add a link `/usr/local/bin/bcompare -> /Applications/Beyond\ Compare.app/Contents/MacOS/bcomp` as "proposed" in https://www.scootersoftware.com/support.php?zz=kb_OSXInstallCLT

Is this on purpose? I didn't find any discussion / issue on this. Any link would be helpful. Thanks!

**Comment by SMillerDev** [comment]
Probably because nobody put in the effort to make it happen.
